homeassistant/components/lutron_caseta/light.py

```diff
@@ -83,3 +83,10 @@ async def async_turn_off(self, **kwargs):
     def is_on(self):
         """Return true if device is on."""
         return self._device["current_state"] > 0
+
+    @property
+    def unique_id(self):
+        """Return a unique identifier if serial number is None."""
+        if self.serial is None:
+            return f"light_{self._bridge_unique_id}_{self.device_id}"
```

Comment:
```suggestion
            return f"{self._bridge_unique_id}_{self.device_id}"
```

light isn't needed since ids are unique per platform